ironfish-cli/src/commands/service/bridge/release.ts

```diff
@@ -210,51 +211,84 @@ export default class Release extends IronfishCommand {
     account: string,
     api: WebApi,
   ): Promise<void> {
-    // TODO(hughy): group multiple requests into a single transaction
-    const nextBurnRequests = await api.getBridgeNextBurnRequests(1)
+    const nextBurnRequests = await api.getBridgeNextBurnRequests(MAX_RECIPIENTS_PER_TRANSACTION)
 
     if (nextBurnRequests.length === 0) {
       this.log('No burn requests')
       return
     }
 
-    const burnRequest = nextBurnRequests[0]
+    const pendingRequests = []
 
-    const response = await client.wallet.getAccountBalance({
-      account,
-      assetId: burnRequest.asset,
-    })
-    const availableBalance = BigInt(response.content.available)
+    const balancesResponse = await client.wallet.getAccountBalances({ account })
+    const availableBalances: Map<string, bigint> = new Map()
+    for (const balance of balancesResponse.content.balances) {
+      availableBalances.set(balance.assetId, BigInt(balance.available))
+    }
 
-    if (availableBalance < BigInt(burnRequest.amount)) {
-      this.log(
-        `Available balance too low to burn ${burnRequest.amount} of asset ${burnRequest.asset}`,
-      )
+    const burnDescriptions: Map<string, BurnDescription> = new Map()
+
+    for (const request of nextBurnRequests) {
+      const assetId = request.asset
+
+      const availableBalance = availableBalances.get(assetId) ?? 0n
+
+      const burnDescription = burnDescriptions.get(assetId) ?? {
+        assetId: Buffer.from(assetId, 'hex'),
+        value: 0n,
+      }
+
+      if (burnDescription.value + BigInt(request.amount) > availableBalance) {
+        continue
+      }
+
+      burnDescription.value += BigInt(request.amount)
+      burnDescriptions.set(assetId, burnDescription)
+      pendingRequests.push(request)
+    }
+
+    if (burnDescriptions.size === 0) {
```

Comment:
Isn't this a huge problem if this happens? Wonder if we should do anything more than log.

Reply:
It might only mean that there are pending transactions that prevent us from sending another one. It is a huge problem if we consistently don't have enough balance because then we've lost some tokens 😰